crypto3/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ find_package(CM REQUIRED)
 include(CMConfig)
 include(CMSetupVersion)
 cm_workspace(crypto3)
-cm_setup_version(VERSION 0.3.0 PREFIX ${CMAKE_WORKSPACE_NAME})
+cm_setup_version(VERSION 0.3.0 PREFIX crypto3)
```

Comment:
why did you replaced `CMAKE_WORKSPACE_NAME` by `crypto3`?

Reply:
It was a topic long ago, back in crypto3 improvements, I just noticed, it is still inconsistent across crypto3/libs and main CMakeLists.
It's much more readable, isn't it? And we don't override the workspace name anywhere